EMS/src/Entities/StudentSolution.java

```diff
@@ -0,0 +1,63 @@
+package Entities;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+
+/**
+ * Represents the Student Solution in a certain question in a certain exam model
+ * It implements SqlEntity to override add() in order to use it in
+ * executing SQL statements and data insertions
+ *
+ * @author Yusuf Nasser, Youssef Nader
+ * @since Mar 22nd, 2021 08:07
+ */
+public class StudentSolution implements SqlEntity {
```

Comment:
In **Java**: **an Interface is implemented** so it's not up to the developer to choose or prefer one instead of the other, you **have to** implement it; otherwise the program would not compile. SqlEntity is an Interface so I have to implement it and override all its methods to use the SQL functionality provided.